main.go

```diff
@@ -21,6 +21,10 @@ import (
 	"go.containerssh.io/libcontainerssh/service"
 )
 
+var (
+	version = "0.5.0"
```

Comment:
This adds an additional layer of complexity to managing the version number as it has to be changed/incremented before a tag and it does not indicate if it was built from git or a in-dev source.

It'd be better to read the info from the [buildinfo[(https://pkg.go.dev/runtime/debug#ReadBuildInfo) API, perhaps of ContainerSSH _and_ libcontainerssh version, which may not always be the same.

Another option would be to use `go generate` to get the version from `git describe`.

Reply:
tried it.
```suggestion
	bi, ok := debug.ReadBuildInfo()
	if !ok {
		return fmt.Errorf("read build info %t", ok)
	}
	for _, dep := range bi.Deps {
		if dep.Path == "go.containerssh.io/libcontainerssh" {
			containersshVersion = dep.Version
		}
	}
```
It works well, but it's still in the devel stage
printed as shown in the image.
![image](https://github.com/ContainerSSH/libcontainerssh/assets/51896679/64b6710f-97e1-4677-acc5-33c150277fd9)

add it if it's officially released.